### ISSUE TITLE
Bookmark title displaying instead of tags

### ISSUE THREAD

**Issue by isaaclo123** [carnager/buku_run#27]
It seems that because buku 4.0 adds a new "title" line to its output, the buku_run script is reading the wrong line for tags, and is displaying the title in place of tags in the menu.

![buku-tag-issue](https://user-images.githubusercontent.com/9246180/50553768-1929aa00-0c62-11e9-8357-e8520e521561.png)

**Comment by plum** [comment]
Yes there is a problem of this sort. In fact problem occurs when editing. Instead of tags,  the the  URl is displayed, and the title is displayed instead of URL; upon exiting the edit URL dialogue the warning 'NOt a valid url ...' is displayed because it is in fact not aa url, but the title.
The title displays "waiting for input" - it is not being read.

So at this time the script doesnt work, except to launch a url ( in elinks in my case, not the xdg default).

I htinkit would be a brilliant boon if it worked again - the two buku and buku_run go so well together.

**Comment by isaaclo123** [comment]
@plum I made a fork that shows the bookmark tag correctly. It seems that GandaG already made a pull request for it but I didn't see it. :P I think mine may be a little more up to date with the master, so feel free to try this fork out; or just use GandaG's fork in #24.

**Comment by plum** [comment]
Thanks I shall try that.
